tests/system/action/user/test_update.py

```diff
@@ -2421,6 +2421,39 @@ def test_update_user_limit_okay(self) -> None:
         self.assert_model_exists("user/4", {"is_active": True})
         self.assert_history_information("user/4", ["Set active"])
 
+    def test_update_clear_user_sessions(self) -> None:
+        self.create_meeting()
+        self.set_models(
+            {
+                "user/2": {
+                    "is_active": True,
+                    "default_password": "no_password",
+                    "username": "user2",
+                    "password": self.auth.hash("no_password"),
+                },
+            }
+        )
+        self.set_user_groups(2, [2])
+        self.login(2)
+        response = self.request(
+            "user.update",
+            {
+                "id": 2,
+            },
+        )
+        self.assert_status_code(response, 200)
+        self.assert_logged_in()
+        response = self.request(
+            "user.update",
+            {
+                "id": 2,
+                "is_active": False,
+            },
+        )
+        self.assert_status_code(response, 200)
+        self.assert_model_exists("user/2", {"is_active": False})
+        self.assert_logged_out()
```

Comment:
It would probably be better to just add the upper half of this test to another test, to ensure that we can differentiate what exactly is failing more easily

Reply:
Since we have a lot of tests and our current test setup creates a relatively big overhead for initializing and cleaning up after the tests, it would be better to keep those similar checks in one test. @rrenkert told me also that the situation is not going to change in the near future.

Reply:
Moved the test case to `test_update_no_OML_set`.